docs/sources/administration/configuration.md

```diff
@@ -1159,6 +1159,24 @@ across larger clusters at the expense of increased bandwidth usage. The default
 
 The interval string is a possibly signed sequence of decimal numbers, followed by a unit suffix (ms, s, m, h, d), e.g. 30s or 1m.
 
+### execute_alerts
```

Comment:
Can we please match the format we have above?

Description, follow by the default in the form of `The default value is XX` and then for options that have a legacy option we can have something like:

- This option has a legacy version in the [respective legacy]({{< relref "#evaluation_timeout_seconds-1">}}) is section that takes precedence.